### PR TITLE
Update/readme

### PR DESCRIPTION
**Pull request by sully90**
### What

Update README and fixes for Dockerfile.base

### How to review

Code review

### Who can review

Anyone but me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,10 @@ To install locally (not recommended), run ```make```. The code requires python3.
 a [virtual environment](https://docs.python.org/3/library/venv.html).
```
Comment (janderson2) [inline README.md]: It looks like the configuration section is out of date.  It is missing the dp-fasttext connection configs and the feature flags.  There may be others.

File: README.md [hunk 2]
```diff
@@ -26,8 +26,10 @@ To install locally (not recommended), run ```make```. The code requires python3.
...
 Alternatively (preferred approach), you can use the supplied Dockerfile to run in a container. When running with 
 conceptual search and user recommendation enabled, the simplest approach is to use ```docker-compose``` with the
-```docker-compose.yml``` provided to bring up dedicated instances of mongoDB and Elasticsearch. Note that for conceptual
-search, the latter requires a [plugin for vector scoring](https://github.com/sully90/fast-elasticsearch-vector-scoring).  
+```docker-compose.yml``` provided (run `docker-compose build` before `docker-compose up`). Note that for conceptual search,
+ Elasticsearch requires a [plugin for vector scoring](https://github.com/ONSDigital/fast-elasticsearch-vector-scoring).
+The `docker-compose.yml` file provided will pull a custom Elasticsearch 2.4.4 image with this plugin pre-installed when 
+working locally (be sure to switch off any instances of Elasticsearch that might already be running first).  
 
 # Running
```
Comment (janderson2) [inline README.md]: Based our README template/standards we should have a brief description of the service, then a getting started section that outlines how to build and run the service locally for development (only the best/recommended approach needs to be here) then the configuration section, and finally the licence details.

File: README.md (hunk 1 above)
Comment (sully90) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (sully90) [reply README.md]: Have updated for second review

**Pull request merged**